app.rb

```diff
@@ -0,0 +1,72 @@
+require_relative "lib/database_connection"
+require_relative "lib/item_repository"
+require_relative "lib/order_repository"
+require_relative "lib/item"
+require_relative "lib/order"
+
+class Application
+  def initialize(database_name, terminal, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @terminal = terminal
+    @item_repository = item_repository
+    @order_repository = order_repository
```

Comment:
I like how you named these instance variables, it's very clear what their purpose is as the name is descriptive.